media-plugins/distrho-ports/distrho-ports-99999999.ebuild

```diff
@@ -22,41 +22,26 @@ REQUIRED_USE="|| ( lv2 vst )"
 
 RDEPEND="media-libs/alsa-lib
```

Comment:
There are some libs listed in the meson.build file that I don't know, any ideas how to determine what package they belong to?
https://github.com/DISTRHO/DISTRHO-Ports/blob/062a0f614703293bbc32eeda0fd65a297604262c/meson.build#L79

Reply:
yeah, dl/rt are standard libc things (dynamic linker and realtime support)

Reply:
Thanks! That was pretty simple, I should've simply checked the libs as you did, thanks for the examples!
Since these are provided by glibc I don't think we need any additional dependencies.